### PR TITLE
Feature/members export csv

### PR DESCRIPTION
**Pull request by mpetrunic**
* adds export to csv action for kset admin section
* enables export to csv action on members

### CODE REVIEW

File: members/actions.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            writer.writerow(field_names)
+        for obj in queryset:
+            row = [__get_field(obj, field)() if callable(__get_field(obj, field)) else __get_field(obj, field) for field in field_names]
+            writer.writerow(row)
+        return response
+    export_as_csv.short_description = description
+    return export_as_csv
+
+
+def make_bill(modeladmin, request, queryset):
```
Comment (morrigan) [inline members/actions.py]: It would be better to use invoice instead of bill.

File: members/actions.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    odd = True
+    cnt = 437
+    for member in queryset:
+        bill.buyer['name'] = smart_str(member.name + " " + member.surname)
+        bill.buyer['taxnum'] = member.id
+
+        bill.info['num'] = "2011-" + str(cnt)
+        bill.info['date'] = "04.01.2011."
+        bill.info['items'] = [['članarina za SSFER', 100.0]]
```
Comment (morrigan) [inline members/actions.py]: Is this the price hardcoded?

File: members/actions.py (hunk 1 above)
Comment (mpetrunic) [inline members/actions.py]: This isn't mine code, I don't this ever worked. Just moved this in separe file with my new action...

File: members/actions.py (hunk 2 above)
Comment (mpetrunic) [inline members/actions.py]: seem like it :D

**Pull request merged**